### PR TITLE
29 mandatory studen4 r02 d02

### PR DESCRIPTION
**Pull request by Naredo**

### CODE REVIEW

File: src/main/webapp/WEB-INF/resources/sample-data/sponsor.csv
Comment (davidg43) [inline src/main/webapp/WEB-INF/resources/sample-data/sponsor.csv:1]: try to use a negative test to be sure about money assert.

**Review by davidg43 — CHANGES_REQUESTED** [review]

**Comment by Naredo** [comment]
Problems fixeds

File: njgsfs
Comment (davidg43) [inline njgsfs:1]: You cant commit this file

File: .settings/org.eclipse.wst.common.component
Comment (davidg43) [inline .settings/org.eclipse.wst.common.component:1]: Its important to resolve this unexpected differences!

**Review by davidg43 — CHANGES_REQUESTED** [review]

**Pull request merged**